app/views/support_interface/provider_users/_provider_user_navigation.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for :title do %>
+  <span class="govuk-caption-xl">Provider user</span>
+  <%= @form.provider_user.full_name %>
+<% end %>
+
+<%= render TabNavigationComponent.new(items: [
+  { name: 'Details', url: edit_support_interface_provider_user_path(@form.provider_user), current: local_assigns[:current] == 'Details' },
+  { name: 'History', url: support_interface_provider_user_audits_path(@form.provider_user), current: local_assigns[:current] == 'History' },
```

Comment:
I don't think the `current:` stuff is necessary here, the `TabNavigationComponent` should automatically highlight the current page (you can use `current:` to highlight tabs that link to a different page, for example if you're on a subpage).

Reply:
Ah, nice! Updated.